### PR TITLE
adding bank holidays to opening hours

### PR DESCRIPTION
**Pull request by philspray**
### Trello card

https://trello.com/c/BPmJTt1T

### Context

Adding an exception for bank holidays to contact opening hours.

### Changes proposed in this pull request
Adding in an exception for bank holidays and a link to gov.uk

### CODE REVIEW

File: app/views/content/non-uk-teachers/international-qualified-teacher-status/_after-accordion.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ This guidance relates to teaching in England. Different rules  apply to [teachin
 
 To get advice about iQTS, you can:
 
-- call +44 800 389 2500, Monday to Friday between 8:30am and 5:30pm (UK local time) – calls will be charged at your standard rate
+- call +44 800 389 2500, Monday to Friday between 8:30am and 5:30pm (UK local time) except on [bank holidays](https://www.gov.uk/bank-holidays) in England – calls will be charged at your standard rate
```
[the remark this answers is not in the document]
Comment (philspray) [reply app/views/content/non-uk-teachers/international-qualified-teacher-status/_after-accordion.md:17]: I went for 'in England' to make it clear the ones in Scotland didn't apply.

File: app/components/content/talk_to_us_component.html.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <div class="talk-to-us-content">
   <h2 class="heading-m heading--box-purple heading--overlap">Talk to us</h2>
-      <p>Get support Monday to Friday between <strong>8:30am and 5:30pm</strong> through live chat or on the phone.</p>
+      <p>Get support through live chat or on the phone, Monday to Friday, <strong>8:30am until 5:30pm</strong> except on <a href=“https://www.gov.uk/bank-holidays”>bank holidays</a>.</p>
```
[the remark this answers is not in the document]
Comment (philspray) [reply app/components/content/talk_to_us_component.html.erb:3]: Sorry, thought I'd fixed that.

File: app/components/content/talk_to_us_component.html.erb [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="talk-to-us-content">
   <h2 class="heading-m heading--box-purple heading--overlap">Talk to us</h2>
-      <p>Get support Monday to Friday between <strong>8:30am and 5:30pm</strong> through live chat or on the phone.</p>
+      <p>Get support through live chat or on the phone, Monday to Friday, <strong>8:30am until 5:30pm</strong> except on <a href="https://www.gov.uk/bank-holidays">bank holidays</a>.</p>
```
Comment (EllieNodder) [inline app/components/content/talk_to_us_component.html.erb:3]: do we need to say what happens on bank holidays, i.e. it's closed?

File: app/components/footer/talk_to_us_component.html.erb [hunk 4]
```diff
@@ -17,7 +17,7 @@
                 <p class="telephone">0800 389 2500</p>
               </div>
             </div>
-            <p>Monday to Friday between <strong>8:30am and 5:30pm</strong></p>
+            <p>Call or chat to us Monday to Friday, <strong>8:30am until 5:30pm</strong>, except on <a href="https://www.gov.uk/bank-holidays">bank holidays</a>.</p>
```
Comment (EllieNodder) [inline app/components/footer/talk_to_us_component.html.erb:20]:
```suggestion
            <p>Call or chat to us online Monday to Friday, <strong>8:30am until 5:30pm</strong>, except on <a href="https://www.gov.uk/bank-holidays">bank holidays</a>.</p>
```
Do we just need to be a bit more explicit here that we're referring to the live chat service, too? (just 'chat to us' could still imply that we're just referring to the phone line)

File: app/views/content/become-a-further-education-teacher.md [hunk 5]
```diff
@@ -31,6 +31,6 @@ You do not need a degree to start teaching in further education.
 
 To find out more, visit the [teach in further education website](https://www.teach-in-further-education.campaign.gov.uk/).
 
-You can also speak to an adviser on [0800 389 2502](tel:0800 389 2502). Lines are open Monday to Friday between 8:30am and 5:30pm.
+You can also speak to an adviser on [0800 389 2502](tel:0800 389 2502). Lines are open Monday to Friday between 8:30am and 5:30pm, except on [bank holidays](https://www.gov.uk/bank-holidays).
```
[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/become-a-further-education-teacher.md:34]: The FE PM has confirmed this is fine to update

**Pull request merged**